packages/client/src/components/app/blocks/RowExplorer.svelte

```diff
@@ -0,0 +1,247 @@
+<script>
+  import Block from "components/Block.svelte"
+  import BlockComponent from "components/BlockComponent.svelte"
+  import { makePropSafe as safe } from "@budibase/string-templates"
+  import { generate } from "shortid"
+
+  export let dataSource
+  export let height
+
+  export let cardTitle
+  export let cardSubtitle
+  export let cardDescription
+  export let cardImageURL
+  export let cardSearchField
+
+  export let detailFields
+  export let detailTitle
+
+  const stateKey = generate()
+
+  let listDataProviderId
+  let listRepeaterId
+</script>
+
+<Block>
+  <BlockComponent
+    type="container"
+    props={{
+      direction: "row",
```

Comment:
I think it would look better with some spacing between the card list and the form block, so maybe we could add `gap: "M"` here.